mypyc/irbuild/for_helpers.py

```diff
@@ -86,6 +86,37 @@ def for_loop_helper(builder: IRBuilder, index: Lvalue, expr: Expression,
     builder.activate_block(exit_block)
 
 
+def for_loop_helper_simple(builder: IRBuilder, index: Lvalue, expr: Expression,
+                           body_insts: Callable[[Value], None], line: int) -> None:
```

Comment:
Add docstring here. Explain all the arguments and how this is different from `for_loop_helper`.